src/__tests__/diagram_generation.test.js

```diff
@@ -23,70 +23,64 @@
  SOFTWARE.
 
  */
-import {single_table_input, single_table_output} from "../test_data/single_table"
+import {database_simple_view_explore, database_simple_view_output} from "./test_data/diagrams_basic"
+import {model_few_views_explore, model_few_views_output} from "./test_data/diagrams_polls"
+import {github_commit_explore, github_commit_dimensions} from "./test_data/github_commit"
+import {github_pr_explore, github_pr_dimensions} from "./test_data/github_pr"
+import {diagrams_gov_explore, diagrams_gov_dimensions} from "./test_data/diagrams_gov"
 import {generateExploreDiagram} from "../utils/LookmlDiagrammer"
-import {polls_all_fields} from "../test_data/polls_all_fields"
-import {polls_all_fields_diagram} from "../test_data/polls_all_fields_diagram"
-import campaign_default from "../test_data/campaign_default.json"
-import campaign_default_diagram from "../test_data/campaign_default_diagram.json"
-import covid_default from "../test_data/covid_default.json"
-import covid_default_diagram from "../test_data/covid_default_diagram.json"
-import ecmap_just_joins from "../test_data/ecmap_just_joins.json"
-import ecmap_just_joins_diagram from "../test_data/ecmap_just_joins_diagram.json"
 
-test('1 table 0 joins - results - default', () => {
-  expect(JSON.stringify(generateExploreDiagram(
-    single_table_input.explore,
-    single_table_input.hiddenToggle,
-    single_table_input.displayFieldType
-  ))).toEqual(JSON.stringify(single_table_output));
+test('lookml_diagrams::database_simple_view', () => {
```

Comment:
These all look similar. You could use a test.each here

See
`packages/api-explorer/src/components/DocSDKs/DocSDKs.spec.tsx` in SDK code gen for an example.